src/content/docs/en/guides/server-side-rendering.mdx

```diff
@@ -365,4 +365,5 @@ const factPromise = fetch('https://catfact.ninja/fact')
 </html>
 ```
 
-In this example, `A name` will render while `personPromise` is loading. At that point, `A fact` will be appear, while `factPromise` is loading.
+In this example, `A name` will render while `personPromise` and `factPromise` are loading.
+Once `personPromise` has resolved `A fact` will be appear and `factPromise` will render when its finished loading.
```

Comment:
I did a small wording change and fixed a typo, let me know what you think!
```suggestion
Once `personPromise` has resolved, `A fact` will appear and `factPromise` will render when it's finished loading.
```

Reply:
Love it, I've added a commit the suggestions. Thanks for the feedback.